src/styles/main.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
```

Comment:
When using `calc()`, there should be spaces around the `+` and `-` operators for better readability. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)